ros/ackermann_vehicle_navigation/launch/track_follower.launch

```diff
@@ -1,9 +1,11 @@
 <launch>
   
   <!-- This node computes the path for the car to follow based on position of cones and car position + direction -->
-  <node name="track_pathfinder" pkg="ackermann_vehicle_navigation" type="track_pathfinder.py" output="screen"/>
+  <node name="track_pathfinder" pkg="ackermann_vehicle_navigation" type="track_pathfinder.py" output="screen" launch-prefix="nice -n -20"/>
 
   <!-- This node compute the transformation required for the car to achieve poses in /path. Output is /cmd_vel -->
-  <node name="path_follower" pkg="ackermann_vehicle_navigation" type="path_follower.py" output="screen"/>
+  <node name="path_follower" pkg="ackermann_vehicle_navigation" type="path_follower.py" output="screen" launch-prefix="nice -n -19"/>
```

Comment:
So we want the path follower at a lower priority than the path finder?

Reply:
And we're using -20 and -19 here, right?

Reply:
Yes